src/configs/test.yml

```diff
@@ -91,4 +91,4 @@ tables:
     time:
       type: macro
       # support granularity between DATE to DATE HOUR MINUTE SECOND
-      pattern: date hour
+      pattern: date HOUR
```

Comment:
I was going to say last time: put multiple token as a pattern name feels strange to me, personal opinion, I think it would be better if we come up with a fixed name, and we can document the mapping, such as 

DAILY -> looking for macro in path {date}
HOURLY -> looking for macro in path {date} and {hour}
MINUTELY -> looking for macro in path {date}, {hour} and {minute}
SECONDLY -> looking for macro in path {date}, {hour}, {minute} and {second}

This form will be consistent as our other supported pattern names like UNIX_TIME, UNIX_TIME_MS, etc.

This is for discussion, do not need to address in this PR.

Reply:
06fdcb7